### ISSUE TITLE
Add widgets toggle button should be clearly marked as a toggle, show toggle state

### ISSUE THREAD

**Issue by Phanoix** [gctools-outilsgc/gcconnex#22]
Add widgets toggle button on all pages should be clearly marked as such and indicate its toggle state, generally done in alt/title text with (toggled) and (not toggled) or something similar.

Some relevant files I've found:
/mod\gc_glee_theme\views\default\page\layouts\widgets\add_button  line ~8+ 
/js\lib\ui.widgets.js
